client/my-sites/checkout/src/hooks/use-tos-foldable-card.tsx

```diff
@@ -7,12 +7,16 @@ import { getSelectedSiteId, getSectionName } from 'calypso/state/ui/selectors';
 
 export function useToSFoldableCard(): boolean {
 	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
-
 	const isJetpackNotAtomic = useSelector(
 		( state ) => isJetpackSite( state, siteId ) && ! isSiteAutomatedTransfer( state, siteId )
 	);
+
+	const section = useSelector( getSectionName );
+	const enabledSections = [ 'business-plan-upgrade-upsell', 'checkout' ];
```

Comment:
Hi @jjchrisdiehl! Thanks for making this change!

I was just curious about why we need an allowlist of sections? Asking because the 1-click checkout modal is shown in a few other places now (plugins page, BBE page picker page), and there is the possibility of it getting added to more pages in the near future. Can we enable it for all sections and remove this condition?